content/api/commands/window.md

```diff
@@ -72,7 +83,7 @@ it('equals bar', () => {
   let foo
 
   cy.window().then((win) => {
-    foo = win.foo
+    foo = win.tags.foo
```

Comment:
If you're going to fix this one, you should also fix the next one on line 104!

Reply:
doh 😄